LTS (#1056)
italia/eidas-it-wallet-docs: docs/en/registry.rst

Reviewer: Is it necessary to specify the need for alt_text for each logo?

Reviewer 2: Currently the CI makes such information available, when present. I would suggest including it in the AS Registry / Registration as OPTIONAL, so as to leave the AS in control of that information.
I am going to add those information.

Reviewer: ```suggestion
.. note::
  For further details on the required features and the expected outcome in terms of user experience, see the section “[Issuance from the Wallet instance Catalog](https://italia.github.io/eid-wallet-it-docs/releases/1.4.0/en/functionalities.html#issuance-from-the-wallet-instance-catalog)” for the parameter `data_capabilities.user_information` and the section “[Focus on Electronic Attestation of Attributes](https://italia.github.io/eid-wallet-it-docs/releases/1.4.0/en/functionalities.html#focus-on-electronic-attestations-of-attributes)” for the parameters `organization_info.logo_uri`, `organization_info.logo_extended_uri`, `data_capabilities.logo_uri`, `data_capabilities.background_image`, `data_capabilities.watermark_image`, `data_capabilities.background_color` and `data_capabilities.available_claims.order`.
```
I suggest this addition to help AS read the TS and better understand the meaning of these parameters.

Reviewer: ```suggestion
       * Road Charging Subscription
```
Telepass is a company, the label should be more generic

Reviewer: ```suggestion
1.  **Accessing the Discovery Endpoint**: The entity (e.g., a Wallet Provider or informational portal) accesses the `Registry Discovery Endpoint` (``.well-known/it-wallet-registry``) to obtain the URI of the **Digital Credentials Catalog** and of the **Taxonomy**.
```
typo

   * - **organization_info.contacts**
     - String Array

>      - REQUIRED. Array of technical/administrative contact email addresses.

Reviewer: Is it through this field that we intend to collect contact details for at least one point of contact for user support, one point of contact for application-related matters and one point of contact for system-related matters (see [PR 1063](https://github.com/italia/eid-wallet-it-docs/pull/1063/changes?diff=unified#diff-cbd5459b99cbcecdb624eeb019d2c5eb83b1898bdf0c6e1e6f4ae30911c0d97e), Appendix E)? If so, I suggest these changes:

```suggestion
     - REQUIRED. Array of contact email addresses for at least one user-support, one application, and one systems specialist.
```

   * - **organization_info.logo_uri#integrity**
     - string
     - CONDITIONAL. Cryptographic digest of the logo image resource for integrity verification. REQUIRED if ``logo_uri`` is present. Format: ``{digest_method}-{digest_value}`` (e.g., ``"sha-256-abc123..."``).
   * - **organization_info.logo_extended_uri**
     - string
     - OPTIONAL. URL to the organization's extended logo image.
   * - **organization_info.logo_extended_uri#integrity**
     - string
     - CONDITIONAL. Cryptographic digest of the extended logo image resource for integrity verification. REQUIRED if ``logo_extended_uri`` is present. Format: ``{digest_method}-{digest_value}`` (e.g., ``"sha-256-abc123..."``).
   * - **data_capabilities**
     - JSON Objects Array
     - REQUIRED. Array containing data capability specifications.
   * - **data_capabilities[].dataset_id**
     - string
     - REQUIRED. The unique identifier of the dataset within the scope of the Authentic Source, which MAY be used as a query parameter for the ``GetAttributeClaims`` service.
   * - **data_capabilities[].data_origin_l10n_id**
     - string

>      - OPTIONAL. Localization key referencing the human-readable name of the data origin or department providing the data (e.g., ``authentic_source1.dataset1.origin``).

Reviewer: Is it through this parameter that we expect to be able to fill in the field that currently appears on the front end under the name "Origine dei dati"? If so, I would suggest the following changes:

```suggestion
     - REQUIRED. Localization key referencing the human-readable name of the data origin or department providing the data (e.g., ``authentic_source1.dataset1.origin``).
```

   * - **data_capabilities[].logo_uri#integrity**
     - string

>      - CONDITIONAL. Cryptographic digest of the logo image resource for integrity verification. REQUIRED if ``logo_uri`` is present. Format: ``{digest_method}-{digest_value}`` (e.g., ``"sha-256-abc123..."``).

Reviewer: ```suggestion
     - CONDITIONAL. Cryptographic digest of the logo image resource for integrity verification. REQUIRED if ``logo_uri`` is present. Format: ``{digest_method}-{digest_value}`` (e.g., ``"sha-256-abc123..."``).
  * - **data_capabilities[].logo_alt_text_l10n_id**
     - string
     - OPTIONAL. Alternative text for the organization's logo image.
```
Addition of "Alt_text" specification also to "data_capabilities.logo"

   * - **data_capabilities[].background_color**
     - string
     - OPTIONAL. String value of the background color related to be displayed together with the data.
   * - **data_capabilities[].contacts**
     - String Array

>      - OPTIONAL. Array of customer service contact email addresses.

Reviewer: ```suggestion
     - OPTIONAL. Array of customer service contacts or user support channels (e.g., email address).
```
In order not to limit the options to email contact only

     - 
       * Employment Documents
       * Employment Status

>        * Access

Reviewer: I suggest using "employment affiliation" instead of "access" (too generic) to broaden the scope of this specific credential class, so that it is not limited only to credentials "for access" but also includes potential affiliation credentials (e.g. powers of attorney and mandates)

```suggestion
       * Employment Affiliation
```

       * Access to healthcare bonuses
       * Use of mental health vouchers
       * Use of sports vouchers

>    * - Access

Reviewer: ```suggestion
   * - Employment Affiliation
```
This suggestion only applies if my previous comment is accepted
